dace/transformation/pass_pipeline.py

```diff
@@ -272,12 +322,18 @@ class Pipeline(Pass):
         print('Promoted scalars:', results['ScalarToSymbolPromotion'])
 
     """
-    passes: List[Pass]
+
+    _passes = properties.ListProperty(element_type=Pass,
+                                      default=[],
+                                      desc='List of passes that this pipeline contains')
+    _pass_names = properties.SetProperty(element_type=str,
+                                         default=set(),
+                                         desc='List of passe names that this pipeline contains')
```

Comment:
why leading underscore??

Reply:
This way they're automatically hidden from the UI. I see that they're not in that sense 'private', however, so we can change it to no underscore and instead explicitly hide them in the UI, like the `CATEGORY`.

Reply:
I think the `(debug)` category is already hidden